packages/data-broker/src/__tests__/services/dhis/builders/buildEventsFromDhisEventAnalytics.test.js

```diff
@@ -82,7 +82,7 @@ describe('buildEventsFromDhisEventAnalytics()', () => {
         orgUnitName: 'Haveluloto',
         eventDate: '2020-02-07T14:33:00.000',
         dataValues: {},
-        trackedEntityId: 'tracked_entity_id_dl_household_2',
+        trackedEntityId: 'tracked_entity_id_dl_household_3',
```

Comment:
Since I added `tracked_entity_id_dl_household_2` as a real TEI in the mock data, I had to change this test a little